counter/src/main/java/org/infinispan/counter/logging/Log.java

```diff
@@ -56,13 +54,13 @@
 //   @Message(value = "Unable to reset non-existing counter '%s'.", id = 28011)
 //   void noSuchCounterReset(ByteString counterName);
 
-   @LogMessage(level = WARN)
-   @Message(value = "Interrupted while waiting for the counter manager caches.", id = 28012)
-   void interruptedWhileWaitingForCaches();
+//   @LogMessage(level = WARN)
```

Comment:
did you leave these lines commented for any reason ?

Reply:
yes. we started to comment unused log messages per https://issues.jboss.org/browse/ISPN-7456

Reply:
ok